### DISCUSSION TITLE
how can i specify user input in the prompt?

### DISCUSSION THREAD

**Discussion by dantepalacio** [eyurtsev/kor#286] [Q&A]
I need to specify suffixes in prompt and can't figure out how to specify user input suffixes. I would like the messages to be between <|start_header_id|>user<|end_header_id|>
USER INPUT<|eot_id|>. 

Prompt:
<|begin_of_text|><|start_header_id|>system<|end_header_id|>
    Your goal is to extract structured information from the user's input that
matches the form described below. When extracting information please make
sure it matches the type information exactly. Do not add any attributes that
do not appear in the schema shown below.
{type_description}\n\n
{format_instructions}<|eot_id|><|start_header_id|>user<|end_header_id|>
USER INPUT<|eot_id|>
<|start_header_id|>assistant<|end_header_id|>

**Comment by eyurtsev** [comment]
You can either create a custom chat model that does the appropriate formatting for the model you're using: https://python.langchain.com/docs/modules/model_io/chat/custom_chat_model/

Alternatively, read through `kor` code and pull out the relevant functionality to generate examples and then format them for your chat model.
